docs/layouts/shortcodes/generated/flink_connector_configuration.html

```diff
@@ -86,6 +86,18 @@
             <td>String</td>
             <td>The blacklist contains several time periods. During these time periods, the lookup table's cache refreshing is forbidden. Blacklist format is start1-&gt;end1,start2-&gt;end2,... , and the time format is yyyy-MM-dd HH:mm. Only used when lookup table is FULL cache mode.</td>
         </tr>
+        <tr>
+            <td><h5>partition.custom-listener.#.param</h5></td>
```

Comment:
Do we really need this?

Reply:
We can pass parameters by table dynamic options.